Cargo.toml

```
@@ -86,5 +87,5 @@ test-log = "0.2.16"
 
 # SP1 dependencies
 sp1-core-machine = "=4.0.0"
-sp1-sdk = "=4.0.0"
+sp1-sdk = { version="4.0.0", features = ["native-gnark"] }
```

I'm not entirely sure why, but we've been fixing the sp1 version with `=` everywhere else.  So:
```suggestion
sp1-sdk = { version = "=4.0.0", features = ["native-gnark"] }
```

---

Better safe than sorry :)